src/assets/schemas/DataRegistrationV1Validation.js

```diff
@@ -1,17 +1,681 @@
+      "description": "Please mark the reasons for which you are requesting an Alternative Data Sharing Plan (check all that apply)",
+      "items": {
+        "type": "string",
+        "enum": ["Legal Restrictions", "Informed consent processes are inadequate to support data for sharing for the following reasons:", "The consent forms are unavailable or non-existent for samples collected after January 25, 2015", "The consent process did not specifically address future use or broad data sharing for samples collected after January 25, 2015", "The consent process inadequately addresses risks related to future use or broad data sharing for samples collected after January 25, 2015", "The consent process specifically precludes future use or broad data sharing (including a statement that use of data will be limited to the original researchers)", "Other informed consent limitations or concerns", "Other"]
+      }
+    },
+    "alternativeDataSharingPlanExplanation": {
+      "type": "string",
+      "label": "Explanation of Request",
+      "description": "Explanation of Request"
+    },
+    "alternativeDataSharingPlanFileName": {
+      "type": "string",
+      "label": "Upload your alternative sharing plan (file upload)",
+      "description": "Upload your alternative sharing plan (file upload)"
+    },
+    "alternativeDataSharingPlanDataSubmitted": {
+      "type": "string",
+      "label": "Data will be submitted",
+      "description": "Upload your alternative sharing plan (file upload)",
+      "enum": ["Within 3 months of the last data generated or last clinical visit", "By batches over Study Timeline (e.g. based on clinical trial enrollment benchmarks)"]
+    },
+    "alternativeDataSharingPlanDataReleased": {
+      "type": "boolean",
+      "label": "Data to be released will meet the timeframes specified in the NHGRI Guidance for Data Submission and Data Release",
+      "description": "Data to be released will meet the timeframes specified in the NHGRI Guidance for Data Submission and Data Release"
+    },
+    "alternativeDataSharingPlanTargetDeliveryDate": {
+      "type": "string",
+      "format": "date",
+      "label": "Target Delivery Date",
+      "description": "Target Delivery Date"
+    },
+    "alternativeDataSharingPlanControlledOpenAccess": {
+      "type": "string",
+      "label": "Does the data need to be managed under Controlled or Open Access?",
+      "description": "Does the data need to be managed under Controlled or Open Access?",
+      "enum": ["Controlled Access", "Open Access"]
+    }
+  }
+};
 const func3 = require("ajv/dist/runtime/ucs2length").default;
-const formats0 = { "_items": ["require(\"ajv-formats/dist/formats\").", { "str": "fullFormats" }, ""] }.date;
+const dateRegex = /^\d{4}-(02-(0[1-9]|[12][0-9])|(0[469]|11)-(0[1-9]|[12][0-9]|30)|(0[13578]|1[02])-(0[1-9]|[12][0-9]|3[01]))$/;
+const dateValid = (value) => dateRegex.test(value);
```

Comment:
This is the meat of the fix - this date regex comes from `/src/components/forms/formValidation.js`

Reply:
I think this validation file is automatically generated from `ajv`, if this change is implemented manually will it be overwritten the next time we run `ajv`?

Reply:
Yes, there is a script that we have to run that re-writes this file. Most of the changes here are from that process. This is not a sustainable process and we need to move away from it which is what I'm looking into now.